src/jit/lower.cpp

```diff
@@ -1038,6 +1050,23 @@ void Lowering::LowerArg(GenTreeCall* call, GenTreePtr* ppArg)
         type = TYP_INT;
     }
 
+#if defined(FEATURE_SIMD) && defined(_TARGET_X86_)
+    // For TYP_SIMD12, local vars are massaged in lower to TYP_SIMD16. However, when storing
+    // them on the stack, we must use TYP_SIMD12.
```

Comment:
I believe that this should say "when storing them in the argument area.", as it is not the case that we use `TYP_SIMD12` when storing a (non-arg) local to its home location.